### ISSUE TITLE
Add minimal docs for reason-react-navigation & the example

### ISSUE THREAD

**Issue by MoOx** [reasonml-community/reason-react-native#578]
We should have something on the website. People are searching for this in "Search" bar without result atm.

**Comment by damiendidi** [comment]
Yes i agree with that

**Comment by idkjs** [comment]
I am missing the point on the links in this site comment. @damiendidi what is this? Thank you.

**Comment by MoOx** [comment]
@idkjs spam

**Comment by idkjs** [comment]
Is this still an issue? Should I tackle this?

**Comment by MoOx** [comment]
Yeah this is still an issue. That said, website sources has been moved to https://github.com/reason-react-native/reason-react-native.github.io & navigation has been moved to https://github.com/reason-react-native/reason-react-navigation
I am moving this issue to the website repo but feel free to do something there if you want (maybe add a page or just content in the mentioning the example (https://github.com/reason-react-native/reason-react-navigation-example) we have in https://reason-react-native.github.io/en/docs/example/ (so https://github.com/reason-react-native/reason-react-native.github.io/blob/src/docs/example.md)
